EleksTubeHAX_pio/src/Backlights.cpp

```diff
@@ -1,6 +1,9 @@
 #include "Backlights.h"
 
 void Backlights::begin(StoredConfig::Config::Backlights *config_)  {
+  #ifdef DEBUG_OUTPUT_VERBOSE
```

Comment:
I am a big fan of debug outputs! Best with log levels and sperateted output streams/files. But we only have the serial interface output here in the moment...

So I already implemented 5 new levels of debug messages...

```
//uncomment for Debug logs via serial interface
#define DEBUG_OUTPUT 1
// #define DEBUG_OUTPUT_VERBOSE 1

// #define DEBUG_OUTPUT_TFT 1
// #define DEBUG_OUTPUT_MENU 1
// #define DEBUG_OUTPUT_BUTTONS 1
// #define DEBUG_OUTPUT_MQTT 1

```

More wanted?

I just left all inside, because I am tired of removing them and re-coding them, every time on a debugging mission.
Makes the code really not more "readable", but it helps, to understand the work and data flow while the code is running...

If wanted, I can remove all this new debug messages...